config/initializers/sidekiq.rb

```diff
@@ -10,8 +10,8 @@
 
 # Create oscn throttling strategy
 Sidekiq::Throttled::Registry.add(:oscn, {
-  threshold: { limit: ENV.fetch('OSCN_THROTTLE', 240).to_i, period: 1.minute },
-  concurrency: { limit: ENV.fetch('OSCN_CONCURRENCY', 10).to_i }
+  threshold: { limit: ENV.fetch('OSCN_THROTTLE', 6).to_i, period: 1.minute },
+  concurrency: { limit: ENV.fetch('OSCN_CONCURRENCY', 1).to_i }
```

Comment:
I noticed this had actually already been implemented. We actually want both this and slowweb until we can update slowweb. Slowweb works for things that aren't in workers but we'll have to use this for workers for now.